selection/algorithms/epic_algorithm.py

```diff
@@ -28,6 +28,8 @@ def _calculate_best_indexes(self, workload):
         logging.info('Calculating best indexes EPIC')
         self.workload = workload
         single_attribute_index_candidates = self.workload.potential_indexes()
+        multi_attribute_index_candidates = single_attribute_index_candidates.copy(
```

Comment:
I think that this name does not suit.
What was bad with the old version?

Reply:
Or was it incorrect, because single_attribute_index_candidates should not be overwritten to generate all valid multi column indexes?

Reply:
Exactly.